docs/howtos/install_epinio_on_eks.md

```diff
@@ -8,38 +8,57 @@ title: ""
 
 This how-to was written using the following versions:
 
-* [epinio helm chart - v1.1.0](https://github.com/epinio/helm-charts/releases/tag/epinio-1.1.0)
-* AWS EKS - Kubernetes v1.22
-* [Ingress Nginx- v1.3.0](https://kubernetes.github.io/ingress-nginx/)
-* [Aws Load Balancer Controller - v2.4.2](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/)
-* [Cert Manager - v1.9.1](https://github.com/cert-manager/cert-manager)
+* [epinio helm chart - v1.6.2](https://github.com/epinio/helm-charts/releases/tag/epinio-1.6.2)
+* AWS EKS - Kubernetes v1.22, v1.23***** or v1.24*****
+* [Ingress Nginx - v1.6.4](https://kubernetes.github.io/ingress-nginx/)
+* [Cert Manager - v1.11.0](https://github.com/cert-manager/cert-manager)
+
+:::caution *) Additional requirements for EKS v1.23 and v1.24
+
+
+* From EKS v1.23 there is a need to configure and install an out-of-tree AWS EBS CSI driver as a addon into your EKS cluster. Please refer to this [EKS documentation](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) for more details.
+* Since EKS v1.24, when the `dockershim` CRI support has been dropped and replaced by `containerd` (supports only trusted HTTPS registries by default), there is a need to allow pulling epinio's app container images from Internal Epinio HTTP Registry. The following configuration must be done on all EKS nodes prior deploying an Epinio app:
+
+    ```shell
+    mkdir -p /etc/containerd/certs.d/127.0.0.1:30500
+    cat > /etc/containerd/certs.d/127.0.0.1:30500/hosts.toml <<EOF
+    server = "http://127.0.0.1:30500"
+
+    [host."http://127.0.0.1:30500"]
+      capabilities = ["pull"]
+    EOF
+    ```
+
+    Eventually you may modify the node count and apply [this manifest](https://raw.githubusercontent.com/epinio/epinio/main/scripts/eks-cri-allow-http-registries.yaml) which will do the nodes configuration for you.
```

Comment:
```suggestion
    Instead of doing this manually it should be easier to simply apply [this manifest](https://raw.githubusercontent.com/epinio/epinio/main/scripts/eks-cri-allow-http-registries.yaml) which will do the nodes configuration for you, after editing it to use the correct node count.
```